aten/src/ATen/SparseTensorUtils.h

```diff
@@ -82,7 +85,7 @@ inline LongTensor flatten_indices(const Tensor& indices, IntArrayRef full_size,
       indices_mult_cpu_vec[i] = mult;
       mult *= full_size[i];
     }
-    auto indices_mult_cpu = indices.type().cpu()
+    auto indices_mult_cpu = indices.dispatch_type().cpu()
```

Comment:
Do we have a non Type/TypeProperties replacement for `tensorFromBlob`? Could this site be replaced with it?

Reply:
Not yet.